otx/cli/tools/demo.py

```diff
@@ -100,12 +107,50 @@ def get_predictions(task, frame):
     return item.get_annotations(), elapsed_time
 
 
+def save_frame(frame, path, capture_type, saved_frames):
+    """Saves frames with predictions to saved_frames to to dump it in one time."""
+
+    filename = Path(path).name
+    if capture_type == "VIDEO":
+        saved_frames[filename].append(frame)
+    else:
+        saved_frames[filename] = frame
+
+
+def dump_frames(saved_frames, output_path, capture):
```

Comment:
Can we extract this function to some utils file (preferable is an existing one) to avoid duplication in exportable code?

Reply:
Done